assets/js/components/UI/Form/EDTFDate.jsx

```diff
@@ -0,0 +1,100 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { useFieldArray } from "react-hook-form";
+import { isEDTFValid } from "../../../services/helpers";
+import { Button } from "@nulib/admin-react-components";
+import { useFormContext } from "react-hook-form";
+
+const UIFormEDTFDate = ({ label, name, required, ...passedInProps }) => {
+  const { control, errors, register } = useFormContext();
+  const { fields, append, remove } = useFieldArray({
+    control,
+    name, // Metadata item form name
+    keyName: "useFieldArrayId",
+  });
+
+  return (
+    <div data-testid="dateCreated-wrapper">
+      <fieldset>
```

Comment:
Since there's so much code duplication between here and `UIFormFieldArray` component, is there a way we can use that component?    

Let's think about why we're not using it, and what would need to be adjusted in the `UIFormFieldArray` component to allow its usage?

Reply:
Is it just the custom validation?  If so, maybe we build that into`UIFormFieldArray`

Reply:
I agree that we should keep DRY code.